src/components/utils/isTextOrBinary.ts

```diff
@@ -0,0 +1,418 @@
+// Copyright (c) 2012 Bevry Pty Ltd
```

Comment:
Why are we copying this instead of using the [npm module](https://www.npmjs.com/package/istextorbinary)?